File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { BluefinTecsMerchantPortalSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await BluefinTecsMerchantPortalSDK.test()
    equal(null !== testsdk, true)
  })

})

File: .sdk/tm/dart/test/exists_test.dart
import 'harness.dart';

import '../lib/BluefinTecsMerchantPortalSDK.dart';

void tests() {
  describe('exists', () {
    test('test-mode', (t) async {
      final testsdk = BluefinTecsMerchantPortalSDK.test();
      equal(true, null != testsdk);
    });
  });
}
